### DISCUSSION TITLE
Possible to Store Incorrect Date?  HxInputDate

### DISCUSSION THREAD

**Discussion by StarGazer202424** [havit/Havit.Blazor#830] [Q&A]
Hi

I am trying to use : https://havit.blazor.eu/components/HxInputDate#HxInputDate

I do have a requirement if a user does say put in an invalid such as Feburary 30th, 2024 that I actually don't want to wipe the input. It's okay to possibly flag it as an invalid input validation, but I need to respect the input and keep it as is even if it is not a date or an non-existent date.

I seem like I can only use a Date Time what is probably causing the problem as it can't convert to it.  I don't see another like property that would be like of a "string" that could hold it as a secondary value.

**Comment by hakenr** [comment]
`HxInputDate` does not clear the input when an invalid date is entered; instead, the `ParsingErrorMessage` is displayed. Unfortunately, there is no standard way to access the invalid text unless it is a valid date.

**Reply by StarGazer202424** [reply]
Okay I thought for some reason it was clearing it. Yea I got sort of a weird case were I need the invalid input and wanted to check if it was supported.

I guess I will look into making my own custom control based on the HxInputDate
